kubernetes/apps/tools/glycemicgpt-bot/app/externalsecret.yaml

```
@@ -0,0 +1,91 @@
+  namespace: tools
+spec:
+  refreshInterval: 1h
+  secretStoreRef:
+    kind: ClusterSecretStore
+    name: onepassword-connect
+  target:
+    name: glycemicgpt-bot-secrets
+    creationPolicy: Owner
+  data:
+    - secretKey: DISCORD_BOT_TOKEN
+      remoteRef: { key: glycemicgpt-bot, property: DISCORD_BOT_TOKEN }
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)